wpilibcIntegrationTests/src/PreferencesTest.cpp

```diff
@@ -7,8 +7,7 @@
 
 #include "Preferences.h"  // NOLINT(build/include_order)
 
-#include <stdio.h>
-
+#include <cstdio>
```

Comment:
This uses std::remove(), which is in cstdio (http://en.cppreference.com/w/cpp/io/c/remove)